### PR TITLE
overlay.get: if we're ignoring metacopy=on, ignore it when it's global

### PR DESCRIPTION
**Pull request by nalind**
If we've determined that we need to ignore a `metacopy=on` mount option, ignore it when it's a globally-set option, too.

Should fix RHBZ #1979966.

### CODE REVIEW

File: drivers/overlay/overlay.go [hunk 1]
```diff
@@ -1262,16 +1275,16 @@ func (d *Driver) get(id string, disableShifting bool, options graphdriver.MountO
...
-			} else {
-				logLevel := logrus.WarnLevel
-				if unshare.IsRootless() {
-					logLevel = logrus.DebugLevel
-				}
-				logrus.StandardLogger().Logf(logLevel, "Ignoring metacopy option from storage.conf, not supported with booted kernel")
 			}
 		}
 	}
+	if hasMetacopyOption(optsList) && !d.usingMetacopy {
```
[the remark this answers is not in the document]
Comment (nalind) [reply drivers/overlay/overlay.go]: Done.

**Review by vrothberg — COMMENTED** [review]
LGTM

**Comment by vrothberg** [comment]
@giuseppe @rhatdan PTAL

**Review by giuseppe — APPROVED** [review]
LGTM

**Pull request merged**